core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/filebeat.yml

```diff
@@ -1,26 +1,50 @@
 ---
-- name: Filebeat | Get information about installed packages as facts
+- name: Filebeat as DaemonSet | Get information about helm releases
+  delegate_to: localhost
+  become: false
+  run_once: true
+  environment: { KUBECONFIG: "{{ vault_location }}/../kubeconfig" }
+  command: helm list
+  register: check_filebeat_release
+
+- name: Reinstall Filebeat as DaemonSet from default namespace to custom namespace for "k8s as cloud service"
+  when:
+    - specification.helm_chart_name in check_filebeat_release.stdout
+    - k8s_as_cloud_service is defined
+    - k8s_as_cloud_service
+  block: 
+    - name: Include uninstall task for Filebeat as DaemonSet in default namespace for "k8s as cloud service"
+      include_role:
+        name: filebeat
+        tasks_from: uninstall-filebeat-as-daemonset.yml
+
+    - name: Include install task for Filebeat as DaemonSet in custom namespace for "k8s as cloud service"
+      include_role:
+        name: filebeat
+        tasks_from: install-filebeat-as-daemonset.yml
```

Comment:
Am I correct that if we run `upgrade` a few times with the same version it will be reinstalled anyway? The same question for the left part of PR.

Reply:
No, it is just a fix in order to reinstall from default namespace to custom namespace the daemonset. If we run the upgrade once, the daemonset will reinstall to custom namespace and if we run upgrade once again, the upgrade process will skip since there is the conditions which check only the default namespace